README.rst

```diff
@@ -49,39 +49,23 @@ documentation <https://cloud.mbed.com/docs/latest/mbed-cloud-sdk-python/>`__:
 1. Create an API key in the `Pelion Device Management
    Portal <https://portal.us-east-1.mbedcloud.com/>`__.
```

Comment:
This is now: https://portal.mbedcloud.com

Reply:
Fixed, along with similar reference in the SSL section at the bottom.